src/python/nimbusml/tests/model_summary/test_model_summary.py

```diff
@@ -96,9 +96,11 @@ def test_model_summary(self):
                 [OneHotVectorizer() << categorical_columns, learner])
             train_stream = FileDataStream(train_file, schema=file_schema)
             pipeline.fit(train_stream, label_column)
-            pipeline.summary()
+            try:
+                pipeline.summary()
+            except:
+                assert False, ("Troubled learner: ", learner)
```

Comment:
Why was this modified? It appears that the extra code does not add any functionality.